client/src/outside/components/AssignmentsDialog.vue

```diff
@@ -40,32 +62,74 @@ const ERRORS = {
   NAME_REQUIRED: 'Assignment name is required',
   ID_REQUIRED: 'Assignment externalId is required',
   ID_PATTERN: 'External Id needs to contain letters, numbers, hypens or underscores',
-  SEED_REQUIRED: 'Assignment EHR data seed is required'
+  SEED_REQUIRED: 'Assignment EHR data seed is required',
+  INVALID_TIME: 'Please, enter a valid 24hrs time'
 }
 
 const EDIT_ACTION= 'edit'
 const CREATE_ACTION = 'create'
 
+const PERSONA_DEFAULT_VALUES = {
+  persona: 'Jason',
+  profession: 'doctor',
+  day: 0,
+  time: '08:00'
+}
+
 export default {
   data () {
+    const { persona, profession, day, time } = PERSONA_DEFAULT_VALUES
     return {
       assignmentName: '', externalId: '', ehrRoutePath: '', description: '',
       actionType: '',
       assignmentId: '',
       selectedSeed: '',
       enableExternalIdEdit: true,
       inUseIds: [],
-      showAdvanced: false
+      showAdvanced: false,
+      professions: [
+        { 
+          value: 'doctor',
+          caption: 'Doctor', 
+        },
+        { 
+          value: 'nurse',
+          caption: 'Nurse', 
+        },
+        { 
+          value: 'pharmacist',
+          caption: 'Pharmacist', 
+        }
+      ],
+      persona, profession, day, time
     }
   },
   components: { AppDialog },
   computed: {
     nameValidate () {
       return this.assignmentName.trim() ? undefined :  ERRORS.NAME_REQUIRED
+    },  
+    timeValidate () {
+      if (this.time && this.time.length > 0) {
+        return validTimeStr(this.time) ? null : ERRORS.INVALID_TIME
+      }
+      return null
     },
     seedValidate () {
       return this.selectedSeed.trim() ? undefined :  ERRORS.SEED_REQUIRED
     },
+    personaValidate () {
+      if (this.persona.length === 0) {
+        return 'The persona name is required!'
```

Comment:
No ! need thanks!

Reply:
But see above because we've changed the approach (sorry) and will not make these required.